### DISCUSSION TITLE
Why MagickReadImage() return false

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#2447] [Help]
Im trying to resize the image with magick wand 

My image is corrupted after the MagickReadImage() method.

and MagickReadImage() return MagickFalse

**Comment by snibgo** [comment]
Do you trap errors, and report error messages? What is the error message?

**Reply by programmerarunkumar** [reply]
Getting this

> insufficient image data in file `OriginalImages/Image86.jpeg' @ error/jpeg.c/ReadJPEGImage_/1191

after MagickReadImage() 
checked Image Before MagickReadImage()  , Image is fine
after the  MagickReadImage()  , my Original Image is corrupted 
why ?

**Comment by snibgo** [comment]
So, the problem is in your input file, "OriginalImages/Image86.jpeg". Don't use an image-sharing website. Put it in a zipfile or something.

**Reply by programmerarunkumar** [reply]
> So, the problem is in your input file, "OriginalImages/Image86.jpeg". Don't use an image-sharing website. Put it in a zipfile or something.

this  "OriginalImages/Image86.jpeg" is path of image 
There is no problem with my image ....
I checked my Image , It is fine ...

**Comment by snibgo** [comment]
IM says it has "insufficient image data". I suggest you post a link to that file so we can test it.

**Reply by hellomsg** [reply]
Did you try another picture?

**Reply by programmerarunkumar** [reply]
> Did you try another picture?

Yes, I tried with different picture , but getting that exception

**Reply by programmerarunkumar** [reply]
> IM says it has "insufficient image data". I suggest you post a link to that file so we can test it.

[https://drive.google.com/file/d/1Q3FtPpje6eVJbAP53bLvtvI4VNUZAkhE/view?usp=sharing](url)

**Reply by snibgo** [reply]
The zipfile is 269 MB, which is too large for me to download, sorry.

**Reply by programmerarunkumar** [reply]
> The zipfile is 269 MB, which is too large for me to download, sorry.

Reduced the file size 

[https://drive.google.com/file/d/1dYQ--6IYzG4XKo9xPyHvvp2QX-KbQ8tI/view?usp=sharing](url)
